### PR TITLE
286 improve datasets documentation to highlight pipeline schema requirements

### PR DESCRIPTION
**Pull request by llewellyn-sl**
Address pipeline schema need on Datasets overview page and link to nf-core and Tower Help resources.

### CODE REVIEW

File: docs/datasets/overview.md [hunk 1]
```diff
@@ -6,31 +6,33 @@ description: 'Managing and using datasets in Nextflow Tower.'
 
 ## Overview
 
-The **Datasets** functionality in Nextflow Tower allows the users to store CSV and TSV formatted files within a workspace and use those as an input one or more pipelines.
+The **Datasets** feature in Nextflow Tower allows users to store CSV and TSV formatted dataset files in a workspace, to use as an input one or more pipelines. 
+
+In order for your pipeline to use your dataset as input during runtime, information about the dataset and file format must be included in the `input` parameters of your [pipeline-schema](/pipeline-schema/overview). We recommend using the nf-core tools [schema build](https://nf-co.re/tools/#pipeline-schema) feature to simplify the schema creation process. Commands include an option to validate and lint your schema file according to best practice guidelines from the nf-core community.
```
Comment (ewels) [inline docs/datasets/overview.md:11]: Whilst we typically use `params.input` for @nf-core pipelines, I don't think that it has to be called this for Tower datasets to work. They should work with any parameter name, as long as it's set up correctly in the schema.

Comment (ewels) [reply docs/datasets/overview.md:11]:
```suggestion
In order for your pipeline to use your dataset as input during runtime, information about the dataset and file format must be included in the relevant parameters of your [pipeline-schema](/pipeline-schema/overview). We recommend using the nf-core tools [schema build](https://nf-co.re/tools/#pipeline-schema) tool to simplify the schema creation process. This automatically scaffolds your schema file and provides a web-based interface to fill in the details. Other commands include options to validate and lint your schema file according to best practice guidelines from the nf-core community. 
```

Comment (ewels) [reply docs/datasets/overview.md:11]: Maybe cut this down and link to `docs/pipeline-schema/overview.md` instead?

**Review by ewels — APPROVED** [review]
Minor tweaking suggestions, otherwise LGTM 👍🏻

**Review by gwright99 — APPROVED** [review]
LGTM so long as you implement Phil's comment  about the input parameter.

File: docs/datasets/overview.md [hunk 2]
```diff
@@ -6,31 +6,33 @@ description: 'Managing and using datasets in Nextflow Tower.'
 
 ## Overview
 
-The **Datasets** functionality in Nextflow Tower allows the users to store CSV and TSV formatted files within a workspace and use those as an input one or more pipelines.
+The **Datasets** feature in Nextflow Tower allows users to store CSV and TSV formatted dataset files in a workspace, to use as an input one or more pipelines. 
+
+In order for your pipeline to use your dataset as input during runtime, information about the dataset and file format must be included in the `input` parameters of your [pipeline-schema](/docs/pipeline-schema/overview.md). We recommend using the nf-core tools [schema build](https://nf-co.re/tools/#pipeline-schema) feature to simplify the schema creation process. Commands include an option to validate and lint your schema file according to best practice guidelines from the nf-core community.
```
Comment (ewels) [inline docs/datasets/overview.md:11]:
```suggestion
In order for your pipeline to use your dataset as input during runtime, information about the dataset and file format must be included in the relevant parameters of your [pipeline-schema](/docs/pipeline-schema/overview.md). We recommend using the nf-core tools [schema build](https://nf-co.re/tools/#pipeline-schema) feature to simplify the schema creation process. Commands include an option to validate and lint your schema file according to best practice guidelines from the nf-core community. 
```

**Review by ewels — CHANGES_REQUESTED** [review]
Still need to remove mention of `params.input`

**Comment by llewellyn-sl** [comment]
> Still need to remove mention of `params.input`

Done. Thanks Phil!

File: docs/pipeline-schema/overview.md [hunk 3]
```diff
@@ -1,44 +1,43 @@
...
+The pipeline schema is based on [json-schema.org](https://json-schema.org/) syntax, with some additional conventions. While you can create your pipeline schema manually, we recommmend the use of [nf-core tools](https://nf-co.re/tools), a toolset for developing Nextflow pipelines.
 
-Instead, we recommmend using tools from the [nf-core](https://nf-co.re/) project, which provides a toolset for developing Nextflow pipelines.
-
-Running the `nf-core schema build` command in your pipeline root directory collects your pipeline parameters and gives you interactive prompts about missing or unexpected parameters. If no existing schema file is found, it will create one for you.
+When you run the `nf-core schema build` command in your pipeline root directory, the tool collects your pipeline parameters and gives you interactive prompts about missing or unexpected parameters. If no existing schema file is found, the tool will create one for you.
 
 For more information, please follow [this link](https://nf-co.re/tools/#build-a-pipeline-schema).
```
Comment (llewellyn-sl) [inline docs/pipeline-schema/overview.md:31]:
```suggestion
For more information, see [this link](https://nf-co.re/tools/#build-a-pipeline-schema).
```

**Pull request merged**